pyproject.toml

```diff
@@ -67,6 +67,22 @@ docs = [
     "sphinxemoji",
     "nbsphinx",
 ]
+xpu = [
+    "torch==2.1.0.post0",
+    "torchvision==0.16.0.post0",
+    "intel-extension-for-pytorch==2.1.20+xpu",
```

Comment:
Please double-check https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.30%2bxpu&os=linux%2fwsl2&package=pip